account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/NotificationHandler.java

```diff
@@ -58,104 +59,104 @@ public NotificationHandler(ConfigurationService configurationService) {
     public Void handleRequest(SQSEvent event, Context context) {
         return segmentedFunctionCall(
                 "account-management-api::" + getClass().getSimpleName(),
-                () -> notificationRequestHandler(event, context));
+                () -> notificationRequestHandler(event));
     }
 
-    public Void notificationRequestHandler(SQSEvent event, Context context) {
+    public Void notificationRequestHandler(SQSEvent event) throws NotificationClientException {
```

Comment:
I'm not sure, but I remember @andrew-moores found that removing a parameter from a handler that looked like it was unused caused dynatrace problems, can we check?

Reply:
Had a discussion with Andrew, while it can cause problems, this is not one of the handlers that is affected by this type of change. Deployment was also successful in a dev environment which means dynatrace is not affected by this change 😄

Reply:
The issue with DynaTrace was removing the over-ridden handleRequest method where the BaseHandler class is being extended.  There is a DynaTrace test to check this method exists in the handler classes to ensure DynaTrace will be able to instrument the lambda.  As this class doesn't extend BaseHandler and there is still a handleRequest method with the existing signature this shouldn't impact DynaTrace but we should have a post deployment check in an environment where DynaTrace is deplopyed.